rst/authinfo.rst

```diff
@@ -2,15 +2,15 @@
 
 .. _authinfo:
 
-=============================
- Storing Backend Information
-=============================
+=============================================
+ Storing Backend Information and Credentials
+=============================================
 
 Normally, S3QL reads username and password for the backend as well as
 an encryption passphrase for the file system from the
 terminal. Alternatively, these (and some other backend parameters) may
 be specified in a configuration file specified by the
-:cmdopt:`--authfile` parameter.
+:cmdopt:`--authfile` parameter or from the ``~/.s3ql/authinfo2`` file.
```

Comment:
~/.s3ql/.authinfo2 is the default value for `--authinfo`, so this formulation seems wrong to me.